pyproject.toml

```diff
@@ -44,7 +44,7 @@ pycoingecko = "^2.2.0"
 detecta = "^0.0.5"
 tradingview-ta = "^3.2.3"
 finvizfinance = "^0.14.0"
-statsmodels = "^0.13"
+statsmodels = "0.13.2"
```

Comment:
@jmaslek can we have `statsmodels = ">=0.13.2"` so we don't pin an exact version or this will cause scipy to break?

Reply:
Causes the dep tree to break.